openhands/agenthub/codeact_agent/function_calling.py

```diff
@@ -22,9 +22,11 @@
     MessageAction,
 )
 
-SYSTEM_PROMPT = (
-    """You are a helpful assistant that can interact with a computer to solve tasks."""
-)
+SYSTEM_PROMPT = """You are a helpful assistant that can interact with a computer to solve tasks.
+<IMPORTANT>
+* If user provides a path, you should NOT assume it's relative to the current working directory. Instead, you should explore the file system to find the file before working on it.
+</IMPORTANT>
```

Comment:
Good kind of change, Anthropic does use xml tags so much, even like this ^^.

Reply:
haha - this is heavily inspired by their quickstart: https://github.com/anthropics/anthropic-quickstarts/blob/8f734fd08c425c6ec91ddd613af04ff87d70c5a0/computer-use-demo/computer_use_demo/loop.py#L56-L70